code/android-core-library/src/main/java/com/adobe/marketing/mobile/FileUtil.java

```diff
@@ -13,6 +13,8 @@
 
 import java.io.*;
 
+// TODO: Remove this class when Java version of ConfigurationExtension is deleted
```

Comment:
nit: Can you remove this TODO? 
We will just move this to compatibility artifact as i am not sure if other extensions are using it.